collect_app/src/main/java/org/odk/collect/android/activities/FormListActivity.java

```diff
@@ -32,6 +37,28 @@ protected void sortByStatusDesc() {
     }
 
     @Override
-    protected void setupAdapter(String sortOrder) {
+    protected void setupAdapter() {
```

Comment:
This is a dangerous thing to have because it means concrete subclasses can get away with not having implementations. I think it should be removed.